### ISSUE TITLE
The requested module 'csso' does not provide an export named 'default'

### ISSUE THREAD

**Issue by CMSworker** [1000ch/Sublime-csso#12]
I tried using the CSSO package, thank you for your work on this package.

Windows 10 · Sublime Text 4180 · Node.js v 20.18.0

When clicking on 'CSSO: Minify CSS' in the Command Palette I'm getting this error:
```
file:/// [...] CSSO/csso.mjs:3
import csso from 'csso';
          ^^^^
SyntaxError: The requested module 'csso' does not provide an export named 'default'
```
![csso-error](https://github.com/user-attachments/assets/3ba241f7-c1e3-41ea-ae41-73eb2994e60f)

**Comment by 1000ch** [comment]
@CMSworker Thank you for reporting.  Is this still happening with [the latest release](https://github.com/1000ch/Sublime-csso/releases/tag/v5.0.1)?

**Comment by CMSworker** [comment]
@1000ch The problem has been solved with [release 5.0.1](https://github.com/1000ch/Sublime-csso/releases/tag/v5.0.1), thank you very much
